js-check/6.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Task 6
+ *
+ * @param {Array} array
+ * @param {number} element
+ * @param {number} index
+ * @returns {Array}
+ */
+
+let splicer = (array, element, index) => {
+    array.splice(index * 2, 0, element);
+    return array;
+};
+
+let modifidedArray = (array1, array2) => {
```

Comment:
I like to use reduce, but IMHO it's not the case where it fits.
it's very easy and effective to solve this problem with simple `for` statement